src/utils/helpers.js

```diff
@@ -0,0 +1,32 @@
+// Takes an amount and returns a string representing
+// the amount in euros for speakers of UK English
+export function getUKFormattedEuros(amount) {
+  const options = { style: 'currency', currency: 'EUR' };
+  return Intl.NumberFormat('en-GB', options).format(amount);
+}
+
+// Takes a date string in `YYYY-MM-DD` format and returns a new string
+// based on options format
+export function getUKFormattedDate(dateString, options) {
+  const dateObj = new Date(dateString);
+  return Intl.DateTimeFormat('en-GB', options).format(dateObj);
+}
+
+export function getUniqueMonths(expenseArray) {
+  return expenseArray.reduce((acc, expense) => {
+    let formattedExpenseDate = getUKFormattedDate(new Date(expense.date), {
+      year: '2-digit',
+      month: 'long',
+    });
+
+    if (
+      acc.lentgh !== 0 &&
```

Comment:
Correct! now we will always have at least the `currentMonth` in the `months` array. Thus the condition is no longer needed! Initially, I needed it because I got some errors when no expenses were added.